diff --git a/src/neo_dga.c b/src/neo_dga.c
index b6346c5..c3ab038 100644
--- a/src/neo_dga.c
+++ b/src/neo_dga.c
@@ -38,10 +38,10 @@
 static Bool NEO_OpenFramebuffer(ScrnInfoPtr, char **, unsigned char **, 
 					int *, int *, int *);
 static Bool NEO_SetMode(ScrnInfoPtr, DGAModePtr);
-static void NEO_Sync(ScrnInfoPtr);
 static int  NEO_GetViewport(ScrnInfoPtr);
 static void NEO_SetViewport(ScrnInfoPtr, int, int, int);
 #ifdef HAVE_XAA_H
+static void NEO_Sync(ScrnInfoPtr);
 static void NEO_FillRect(ScrnInfoPtr, int, int, int, int, unsigned long);
 static void NEO_BlitRect(ScrnInfoPtr, int, int, int, int, int, int);
 #if 0
@@ -57,8 +57,8 @@ DGAFunctionRec NEODGAFuncs = {
    NEO_SetMode,
    NEO_SetViewport,
    NEO_GetViewport,
-   NEO_Sync,
 #ifdef HAVE_XAA_H
+   NEO_Sync,
    NEO_FillRect,
    NEO_BlitRect,
 #if 0
